src/Microsoft.VisualStudio.ProjectSystem.Managed/Text/LazyStringSplit.cs

```diff
@@ -37,6 +37,25 @@ public LazyStringSplit(string input, char delimiter)
 
         IEnumerator<string> IEnumerable<string>.GetEnumerator() => GetEnumerator();
 
+        public IEnumerable<T> Select<T>(Func<string, T> func)
```

Comment:
Nice util, I wish this type of helpful utils were shared somewhere across VS for others to reuse ..

Reply:
Where would it live?

Reply:
Where would it live?